geometry/convex_hull.cpp

```diff
@@ -0,0 +1,69 @@
+#include "geometry/convex_hull.hpp"
+
+#include "geometry/robust_orientation.hpp"
+
+#include "base/stl_helpers.hpp"
+
+#include <algorithm>
+
+using namespace std;
+
+namespace m2
+{
+namespace
+{
+double const kEps = 1e-9;
```

Comment:
Please add a comment on why you chose this epsilon (even if the choice was arbitrary).

Reply:
Done